lib/computer_player.rb

```diff
@@ -0,0 +1,24 @@
+class ComputerPlayer
+  attr_reader :mark
+
+  def initialize(mark, board, display)
+    @board = board
+    @display = display
+    @mark = mark
+  end
+
+  def move
+    @display.make_move_message(@mark)
+    index = retrieve_computer_index(1)
+    @display.computer_thinking
+    sleep 1
+    @display.computer_move(index + 1)
+    @board.valid_move?(index) ? @board.mark(index, @mark) : move
+  end
+
+  def retrieve_computer_index(seed)
+    srand(seed)
+    available_move = @board.available_moves
+    available_move[0]
```

Comment:
Does this actually generate a random move? Or does it just return the first available cell in the array of `available_moves`?